dashboard/Chart.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from "react";
```

Comment:
Do you use the chart already? Do you like to keep it for later?

Reply:
Was keeping it to use it later, but let's keep it barebone for now so will remove it